ansys/mapdl/reader/rst.py

```diff
@@ -233,7 +233,16 @@ def _read_result_header(self):
         standard_header = read_standard_header(self.filename)
 
         # Read .RST FILE HEADER
-        header = parse_header(self.read_record(103), result_header_keys)
+        try:
+            record = self.read_record(103)
+            if record.size > 80:
+                raise RuntimeError("Invalid record size. Should be 80 entries")
+            header = parse_header(record, result_header_keys)
+        except:
```

Comment:
Do we know the expected exception? We should avoid bare exceptions as much as possible.

Reply:
Cleaned up the error in [7a3a5b8](https://github.com/ansys/pymapdl-reader/pull/474/commits/7a3a5b81bd4ba4b3521732b8003d52815365cfdb).